### ISSUE TITLE
pretrained weights for Faster-RCNN

### ISSUE THREAD

**Issue by zcablii** [mitming/AlignDet#30]
Thank you for your excellent work!
It seems that the link provided in the README for the pretrained Faster-RCNN weights is not correct as it links to mask-rcnn weights. Could you kindly update the links or provide an alternative way to access the weights?

**Comment by liming-ai** [comment]
> Thank you for your excellent work! It seems that the link provided in the README for the pretrained Faster-RCNN weights is not correct as it links to mask-rcnn weights. Could you kindly update the links or provide an alternative way to access the weights?

Hi @zcablii. Sorry for the late reply. The Faster R-CNN and the Mask R-CNN has the same pretrained weights since we only pretrain the detection head. During fine-tuning stage, Mask R-CNN's mask head is initialized randomly and the detection head is pretrained. Please note here the only difference between Mask R-CNN and Faster R-CNN is whether has a mask head
